operator/pkg/central/values/translation/pvc_checker.go

```diff
@@ -0,0 +1,38 @@
+package translation
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/pkg/utils"
+	corev1 "k8s.io/api/core/v1"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
+	ctrlClient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// pvcExistenceChecker wraps up the information and function to detect
+type pvcExistenceChecker struct {
+	ctx         context.Context
+	client      ctrlClient.Client
+	obsoletePvc bool
```

Comment:
I'm not sure this bool should conceptually belong to this checker object. It does not seem to affect its key responsibility (doing a `client.Get`) in any way...